modules/main-rules/pages/transport.adoc

```diff
@@ -5,35 +5,31 @@ We explain special abilities in more detail at xref:special-rules:transport-x.ad
 
 Units that have the {transport} special ability can carry other units across the battlefield.
 
-* To embark or disembark other units (or cargo) it costs the {transport} unit 5cm of its movement.
+* To embark or disembark other units (or cargo) costs the {transport} unit 5cm of its movement.
 * An embarking or disembarking unit must also pay 5cm of its movement.
 * You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
-* You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example. 
+* You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
 Obviously, once you disembark them then the normal rules apply again.
```

Comment:
I just remembered some discussions about how to measure movement when disembarking, and whether you should place the unit then measure, or measure and then place the unit.

The main note is a bit wordy, let me know if you can think of something better.

```suggestion
Obviously, once you disembark them then the normal rules apply again.

NOTE: Measure the movement of disembarking units from the base of the {transport} unit. If there is not enough movement to clear the {transport} unit's base, place the disembarking unit in base contact with the {transport} unit.

[NOTE.e40k]
====
This was not specified in _Epic 40,000_, but reduces the effects of different size bases.
====
```

Reply:
I don't 100% understand what you're trying to say here, so may be good to talk it through. I'm sure we've discussed it before, but I've forgotten the crux of it.  I intend to make some other edits here anyway, so I'll push those later and then we can hone it some more to get across the exact meaning you intend in the best way.

Reply:
The crux of it is that if you place a disembarking unit in base contact with the Transport, and **then** measure the disembarking unit's movement, different base sizes have a substantial effect on how far the unit can move. 

For example a unit on a 20mm square base unit will lose a couple of centimetres of movement compared to a unit on a 40x12.5mm strip base.

Reply:
I'll resolve this conversation as I've used this suggestion as the core for changes which now appear below.

Reply:
>The crux of it is ...

Lol, I think you replied before I posted my message to say I would mark resolved because the new content is below. I'll unresolve so I have a prompt to look at it again. 

Gotcha, I remember now. I'll wordsmith a little more to make this more explicit.

Reply:
That looks good, resolving the conversation!

Reply:
Yeah, I didn't get to further word-smithing yet to make that point more explicit. But now that you've reminded me of the crux of the point and I look at the word-smithing I already did, I agree that it is probably adequate and we can get away with not making it more explicit.

Though frankly I don't know why anyone would ever place the unit and then measure. To me, that is obviously wrong -- as wrong as measuring from the front of the base to the back where again you'd gain the width of the base.

Something further occurs to me. There appears to be no reason why you couldn't clear the transport unit's base - you must hold back enough movement to do it, on top of the 5cm cost for embarking/disembarking. We could make *that* explicit. But where the problem really lies IIRC is when you have *lots* of units such that potentially you can't fit them all around the transport (Orkeosaurus). In that case, it isn't clearing the base of the transport that is the problem, it's running out of space even when you've tried to do it all right, isn't it?

Reply:
P.S. As far as the actual pull request is concerned, I think this can be marked as resolved.

Reply:
Some interesting points, and definitely room for some more nuance here that we might want to explain eventually. But I agree, let's resolve this for now and revisit in future if and when we feel the urge.